recipes/mysql.rb

```diff
@@ -18,8 +18,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-include_recipe "mysql::client"
```

Comment:
because it's not needed :-)

Probably because [`mysql-server-5.5`](https://packages.debian.org/jessie/mysql-server-5.5) debian package has `mysql-client-5.5` as dependency.